### ISSUE TITLE
UI - ArcGIS Feature Layer Errors

### ISSUE THREAD

**Issue by ewlarson** [BTAA-Geospatial-Data-Project/geoportal#278]
https://geo.btaa.org/catalog/c7074054e2324e5cbd5688766789b4eb_0

Clicking points on the map throw a JS error.

<img width="1280" alt="Screen Shot 2019-11-19 at 4 56 43 PM" src="https://user-images.githubusercontent.com/69827/69193977-a03b7700-0aed-11ea-8e12-cb2fab151f2a.png">

**Comment by karenmajewicz** [comment]
Throwing this one in there:
https://geo.btaa.org/catalog/ab0f0408fcb5479e97df5a13d9a3c648_0
Querying these points also does not work, although I am not seeing a JS error. Maybe different problem?

**Comment by ewlarson** [comment]
https://geo.btaa.org/catalog/c7074054e2324e5cbd5688766789b4eb_0

```json
{OBJECTID: 1379986, ConstructionType: "BRIDGE CLOSED", DetourRoute: null, EndDate: 2556079200000, ID: "{CC68BB52-75E5-4AB4-9EB0-8A1BCBDFD4FB}", …}
```

The "DetourRoute" key returns null and that makes GBL's esri.js code error and ultimately fail to populate the attribute table.

<img width="1280" alt="Screen Shot 2019-11-19 at 5 29 18 PM" src="https://user-images.githubusercontent.com/69827/69195749-307bbb00-0af2-11ea-8659-b84f0d4cb37a.png">
